src/main/java/mate/academy/Main.java

```diff
@@ -6,16 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
```

Comment:
why do we need this initialization?

Reply:
![image](https://user-images.githubusercontent.com/45822497/171016123-7d57877a-200a-4714-8f1f-d0843ccfd988.png)
there is a compilation error without initialization